lib/hyperspec.el

```diff
@@ -137,7 +137,8 @@ If you copy the HyperSpec to another location, customize the variable
     (forward-line)))
 
 (defun common-lisp-hyperspec--parse-map-file (file)
-  (with-current-buffer (find-file-noselect file)
+  (with-temp-buffer
+    (insert-file-contents-literally file)
```

Comment:
Maybe `insert-file-contents` would be safer here, should the symbol table file not be in the same encoding as the default buffer encoding? E.g. if the symbol table is in CRLF format, symbol names will be parsed with a trailing CR.

Reply:
Yeah, that's a bug - fixed in 44126138b24a08b665aacc981be09fd37dd8746f